app/components/round-tracker.js

```diff
@@ -0,0 +1,94 @@
+// https://github.com/EnCiv/civil-pursuit/issues/101
+'use strict';
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import StatusBadge from './status-badge';
+
+const RoundTracker = ({ roundsStatus = [], isMobile = false }) => {
+  const classes = useStyles({ isMobile });
+
+  const renderRounds = () => {
+    let visibleRounds;
+
+    if (isMobile) {
+      const currentRoundIndex = roundsStatus.indexOf('inProgress');
+      if (currentRoundIndex === 0) {
+        visibleRounds = roundsStatus.slice(0, 2); // Show the first two rounds
+      } else if (currentRoundIndex === roundsStatus.length - 1) {
+        visibleRounds = roundsStatus.slice(currentRoundIndex); // Show only the last round
+      } else {
+        visibleRounds = roundsStatus.slice(currentRoundIndex, currentRoundIndex + 2); // Show the current and next rounds
+      }
+    } else {
+        const currentRoundIndex = roundsStatus.indexOf('inProgress');
+      if (currentRoundIndex === 0) {
+        visibleRounds = ['inProgress', 'pending', 'pending'];
+      } else if (currentRoundIndex < 11) {
```

Comment:
Shouldn't use a hard coded number here. Why not roundStatus.length -1?  Is there a reason to have an absolute maximum?

If so (and I hope not) then you should do something like this at the top
```
const MAX_ROUNDS=11 // an explanation about why this constant is needed
```